azure.yaml

```diff
@@ -1,4 +1,24 @@
 - active: false
+  authorities:
+  - endpoint: http://app-operator:8000
+    name: app-operator
+    version: 1.0.0
+  - endpoint: http://azure-operator:8000
+    name: azure-operator
+    version: 2.8.0
+  - endpoint: http://cert-operator:8000
+    name: cert-operator
+    version: 0.1.0
+  - endpoint: http://chart-operator:8000
+    name: chart-operator
+    version: 0.7.0
+  - endpoint: http://cluster-operator:8000
+    name: cluster-operator
+    provider: azure
+    version: 0.21.0
```

Comment:
@tfussell @MarcelMue 

cluster-operator 0.22.0 was wired in https://github.com/giantswarm/cluster-operator/pull/752 

Can we use it here?

Reply:
Yes, we should update this to 0.22.0